.github/workflows/sanity.yaml

```diff
@@ -18,3 +17,13 @@ jobs:
           go-version: '~1.17'
       - name: Run verification checks
         run: make verify
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '~1.17'
+      - name: Run verification checks
+        run: make lint
```

Comment:
Let's use the golangci-lint first-class action for now as it caches the module cache under-the-hood which speeds up linting runs significantly. You can check out https://github.com/operator-framework/operator-lifecycle-manager/blob/master/.github/workflows/sanity.yaml#L23-L25 for a concrete use in OLM.

Reply:
We can also remove any Makefile targets that are covered by the golangci-lint integration (e.g. anything that runs `gofmt ...` or `goimports ...`, etc.).